web/styles/search.css

```diff
@@ -2,144 +2,219 @@
     width: 100%;
     height: var(--header-height);
 
+    .navbar-search {
+        margin: 5.5px 0;
+        border-radius: 5px;
+        position: absolute;
+        color: hsl(0deg 0% 0% / 50%);
+        background: var(--color-background-search);
+    }
+
+    .search_icon {
+        text-decoration: none;
+        padding: 0 7px;
+        font-size: 17px;
+        border: none;
+        background-color: transparent;
+        color: hsl(0deg 0% 0%);
+        opacity: 0.5;
+        cursor: default;
+
+        &:active,
+        &:focus {
+            outline: none;
+        }
+
+        &:disabled {
+            visibility: hidden;
+        }
+    }
+
+    .search_close_button {
+        width: 28px;
+        background: none;
```

Comment:
There's a lot of reset-looking stuff here. Again, the browser inspector isn't playing nice for me to look at this, but it might be worth seeing if any of these can be safely removed.

Reply:
Ah yeah this is blindly removing default button styles, but they're not all needed here. I just removed about half of them!